Author: Reiner Herrmann <reiner@reiner-h.de>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=803412
Description: Use UTC to get date independent of timezone
Last-Updated: 2015-10-29
Forwarded: no

Index: lcov-1.12/bin/updateversion.pl
===================================================================
--- lcov-1.12.orig/bin/updateversion.pl
+++ lcov-1.12/bin/updateversion.pl
@@ -83,7 +83,7 @@ sub get_file_info($)
 
 	return (0, 0, 0) if (!-e $filename);
 	@stat = stat($filename);
-	($sec, $min, $hour, $day, $month, $year) = localtime($stat[9]);
+	($sec, $min, $hour, $day, $month, $year) = gmtime($stat[9]);
 	$year += 1900;
 	$month += 1;
 
